cafebabel/templates/articles/proposals/create.html

```diff
@@ -42,11 +46,11 @@ <h3>Wanna be part of Cafébabel project? That’s the right place</h3>
           </select>
         </label>
       </div>
-    <label for=field_additional>Additional informations</label><textarea name=additional id=field_additional></textarea>
+    <label for=field_additional>Additional information</label><textarea name=additional id=field_additional></textarea>
 
-    <button type=submit>Suggest this article</button>
+    <button type=submit>You're all done! Submit your pitch</button>
```

Comment:
I'd use a `’` instead of the `'` to be consistent.

Reply:
ok !